app/controllers/BulkController.scala

```diff
@@ -55,27 +56,34 @@ class BulkController @Inject()(authAction: AuthAction,
 
   def getPreviousRequests(userId: String) = authAction.async {
     implicit request =>
+    {
+      Logger.debug(s"Request received: ${request}")
```

Comment:
Do we need to log this?

Reply:
To escape 'unused variable'. But if I use "_ =>" instead of "implicit request =>" and if works, will that be okay?